lib/config.js

```diff
@@ -107,7 +108,7 @@ const baseConfig = {
 }
 
 const createConfig = (opts = {}) => {
-  const config = Object.assign({}, baseConfig)
+  const config = merge(baseConfig, opts.webpack)
```

Comment:
I can't remember, but do you know if this clones the `baseConfig`?

Reply:
Looks like it: https://github.com/survivejs/webpack-merge/blob/master/src/index.js#L15